packages/core/src/Stores/client-store.js

```diff
@@ -2729,6 +2731,18 @@ export default class ClientStore extends BaseStore {
         }
     }
 
+    passkeysTrackActionEvent({ action, subform_name, error_message }, is_effortless_modal = false) {
```

Comment:
i see this is just a common function and has nothing to do with stores or observables. can we move it out of stores?
or is there any usecase i miss here?